pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -15,10 +15,17 @@ const (
 	MachineValid                status.ConditionType = "MachineValid"
 )
 
+// AllConditionTypes is a operator conditions currently in use, any condition not in this list is not
+// added to the operator.status.conditions list
 func AllConditionTypes() []status.ConditionType {
 	return []status.ConditionType{InternetReachableFromMaster, InternetReachableFromWorker, MachineValid}
 }
 
+// ClusterChecksTypes represents checks performed on the cluster to verify basic functionality
+func ClusterChecksTypes() []status.ConditionType {
+	return []status.ConditionType{InternetReachableFromMaster, InternetReachableFromWorker, MachineValid}
+}
+
```

Comment:
I dont understand yet why this needs to be copied, but willing to see in follow-up PRs.